virttest/utils_test/__init__.py

```diff
@@ -402,6 +402,12 @@ def run_image_copy(test, params, env):
     2) Check the existence of source image
     3) If it exists, copy the image from NFS
 
+    Note about 'source_image_name' parameter.  'image_copy' test is used for
+    copying existing/prestine image for usage in further tests.
```

Comment:
I think you meant **pristine**, right? I'm not native, but the only meaning of "prestine" that I've found does nor seem to match the intention here. "Pristine", on the other hand, matches perfectly.